contents/docs/session-replay/_snippets/android-installation.mdx

```diff
@@ -0,0 +1,46 @@
+> 🚧 **NOTE:** Android is considered `beta` and is free while in beta. We are keen to gather as much feedback as possible so if you try this out please let us know. You can send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Asession_replay%3Alow) or one of our other [support options](/docs/support-options).
+
+## Installation 
+
+[Install and configure PostHog](/docs/libraries/android#installation) as you normally would. Ensure you've enabled session recordings in your [project settings](/docs/session-replay/installation#step-two-enable-session-recordings-in-your-project-settings) and then add `sessionReplay = true` to your PostHog configuration alongside any of your other configuration options.  
```

Comment:
Since its now in the install step, instead of linking somewhere else on how to install, lets put the instructions directly here like how we do for the other products
e.g. https://posthog-okn6j3jge-post-hog.vercel.app/docs/product-analytics/installation?tab=Android

We should do the same for the other platforms too